config/routes.rb

```diff
@@ -6,8 +6,12 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-   root "users#index"
+  
+   post 'likes', to: 'likes#create'
```

Comment:
- [ ] [MANDATORY]  I noticed that you had made the correct changes in [this commit](https://github.com/francksefu/blog_web/pull/7/commits/df68e679f0e53bec19a0fc0431336b9dca7ebf79) then deleted it again as requested by the previous reviewer [here](https://github.com/francksefu/blog_web/pull/7#discussion_r1460487689). Please ensure that you re add this to the resources block of the posts routes. JUst below comments. In completion it would look like this:

```
resources :posts, only: [:index, :show, :new, :create] do
      resources :comments, only: %i[new create]
      resources :likes, only: [:create]
end
```